test/highlight/c/regular.c

```diff
@@ -2,6 +2,16 @@
 
 #define MACRO 0
 
+int c_init() { 1 }
```

Comment:
I guess you want to return a fixed number?

```suggestion
int c_init() { return 1; }
```

Reply:
That seems reasonable, yes.